### PR TITLE
Issue171

### PR DESCRIPTION
**Pull request by danmarg**

### CODE REVIEW

File: mta-sts.txt [hunk 1]
```diff
@@ -375,11 +375,11 @@ Internet-Draft                   MTA-STS                     August 2017
 
    Parsers MUST accept TXT records and policy files which are
    syntactically valid (i.e. valid key/value pairs separated by semi-
-   colons for TXT records) and implementing a superset of this
-   specification, in which case unknown fields SHALL be ignored.  If any
-   field other than "mx" is duplicated, the first entry will be honored,
-   the rest should be ignored.  For the "mx" field, all valid entries
-   will be utilized when enforcing the stated policy.
+   colons for TXT records) and but containing additional key/value pairs
```
Comment (aykevl) [inline mta-sts.txt]: "and but" - typo?

File: mta-sts.txt [hunk 2]
```diff
@@ -251,29 +251,29 @@ sts-ext-value   = 1*(%x21-3A / %x3C / %x3E-7E)       ; chars excluding
 
 3.2.  MTA-STS Policies
 
-   The policy itself is a set of key/value pairs served via the HTTPS
-   GET method from the fixed [RFC5785] "well-known" path of ".well-
-   known/mta-sts.txt" served by the "mta-sts" host at the Policy Domain;
-   the [RFC2616] "Content-Type" header MUST be "text/plain".  Thus for
-   "example.com" the path is "https://mta-sts.example.com/.well-known/
-   mta-sts.txt".
+   The policy itself is a set of key/value pairs (similar to [RFC2822]
```
Comment (aykevl) [inline mta-sts.txt]: RFC2822 is obsoleted by [RFC5322](https://tools.ietf.org/html/rfc5322) - sorry hadn't checked that before.

File: mta-sts.txt (hunk 1 above)
Comment (danmarg) [inline mta-sts.txt]: Good catch. But I overwrote this in a later commit. :)

File: mta-sts.txt (hunk 2 above)
Comment (danmarg) [inline mta-sts.txt]: Yep, got it.

**Pull request merged**